assets/xml/objects/object_mu.xml

```diff
@@ -1,62 +1,63 @@
-        <Limb Name="object_mu_Standardlimb_00B254" Type="Standard" EnumName="OBJECT_MU_LIMB_14" Offset="0xB254" />
-        <Skeleton Name="object_mu_Skel_00B2B0" Type="Flex" LimbType="Standard" LimbNone="OBJECT_MU_LIMB_NONE" LimbMax="OBJECT_MU_LIMB_MAX" EnumName="ObjectMuLimb" Offset="0xB2B0" />
-        <Animation Name="object_mu_Anim_00BAC4" Offset="0xBAC4" />
+        <Animation Name="gHoneyAndDarlingCupCheeksLoopAnim" Offset="0x1F74" />
+        <Animation Name="gHoneyAndDarlingHugLoopAnim" Offset="0x2F64" />
+        <Animation Name="gHoneyAndDarlingGameDanceLoopAnim" Offset="0x4904" />
+        <Animation Name="gHoneyAndDarlingHoldHandsLoopAnim" Offset="0x5304" />
+        <Animation Name="gHoneyAndDarlingIdleAnim" Offset="0x53E0" />
+        <Texture Name="gHoneyAndDarlingTLUT" OutName="honey_and_darling_tlut" Format="rgba16" Width="16" Height="16" Offset="0x53F0" />
+        <Texture Name="gHoneyAndDarlingShirtsAndShoesTex" OutName="honey_and_darling_shirts_and_shoes" Format="i8" Width="16" Height="16" Offset="0x55F0" />
+        <Texture Name="gHoneyAndDarlingWomanDressTex" OutName="honey_and_darling_woman_dress" Format="ci8" Width="16" Height="16" Offset="0x56F0" />
+        <Texture Name="gHoneyAndDarlingJointShadeTex" OutName="honey_and_darling_joint_shade" Format="i8" Width="8" Height="8" Offset="0x57F0" /> <!-- For the man's shoulders, elbows, and knees, and the woman's elbows -->
+        <Texture Name="gHoneyAndDarlingHeadSkinTex" OutName="honey_and_darling_head_skin" Format="ci8" Width="8" Height="8" Offset="0x5830" /> <!-- For their necks, under chin, noses, cheeks, and foreheads -->
+        <Texture Name="gHoneyAndDarlingManHairTex" OutName="honey_and_darling_man_hair" Format="ci8" Width="16" Height="16" Offset="0x5870" /> 
+        <Texture Name="gHoneyAndDarlingEarTex" OutName="honey_and_darling_ear" Format="ci8" Width="8" Height="8" Offset="0x5970" />
+        <Texture Name="gHoneyAndDarlingMouthTex" OutName="honey_and_darling_mouth" Format="ci8" Width="32" Height="16" Offset="0x59B0" />
+        <Texture Name="gHoneyAndDarlingManEyeTex" OutName="honey_and_darling_man_eye" Format="ci8" Width="32" Height="32" Offset="0x5BB0" />
+        <Texture Name="gHoneyAndDarlingHandTex" OutName="honey_and_darling_hand" Format="ci8" Width="8" Height="8" Offset="0x5FB0" />
+        <Texture Name="gHoneyAndDarlingWomanHairTex" OutName="honey_and_darling_woman_hair" Format="ci8" Width="8" Height="16" Offset="0x5FF0" />
+        <Texture Name="gHoneyAndDarlingWomanEyeTex" OutName="honey_and_darling_woman_eye" Format="ci8" Width="32" Height="32" Offset="0x6070" />
+        <DList Name="gHoneyAndDarlingLegsDL" Offset="0x9370" />
+        <DList Name="gHoneyAndDarlingWomanHeadDL" Offset="0x96C8" />
+        <DList Name="gHoneyAndDarlingWomanRightHandDL" Offset="0x9BC8" />
+        <DList Name="gHoneyAndDarlingWomanRightLowerArmDL" Offset="0x9D10" />
```

Comment:
Just picking on this one since it's first: is this the forearm, or something else? We tend to prefer "forearm" for humanoids unless the limb is weird for some reason.

Reply:
Yes, they're the forearms. Thank you for pointing that out!